src/middlewares/adjustAudio.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable */
+const path = require("path");
+const ensureDir = require("../util/ensureDir");
+const removeDir = require("../util/removeDir");
+const cutAudio = require("../services/cutAudio");
+const extractStartTime = require("../services/extractStartTime");
+const { TEMP_DIR_WAV, TEMP_DIR_WAV_CUT, TEMP_DIR_WAV_ADJUSTED } = require("../constants/paths");
+
+async function adjustAudio(req, res, next) {
+  try {
+    const videoLabels = Object.keys(req.body);
+    const audioPaths = {};
+
+    for (let i = 0; i < videoLabels.length; i += 1) {
+      const videoLabel = videoLabels[i];
+      const startPoint = req.body[videoLabel];
+
+      let inputPath = "";
+      let outputPath = "";
+      let adjustedOutputPath = ""
+
+      if (startPoint === "") {
+        continue;
+      }
+
+      switch(videoLabel) {
+        case "mainStartPoint":
+          adjustedOutputPath = path.join(TEMP_DIR_WAV_ADJUSTED.MAIN, "main-audio.wav");
+          inputPath = path.join(TEMP_DIR_WAV.MAIN, "main-audio.wav");
+          outputPath = path.join(TEMP_DIR_WAV_CUT.MAIN, "main-audio.wav");
+          audioPaths.mainAudio = { outputPath, adjustedOutputPath};
+          ensureDir(TEMP_DIR_WAV_CUT.MAIN);
+          ensureDir(TEMP_DIR_WAV_ADJUSTED.MAIN);
+          break;
+
+        case "subOneStartPoint":
+          adjustedOutputPath = path.join(TEMP_DIR_WAV_ADJUSTED.SUB_ONE, "sub-one-audio.wav")
+          inputPath = path.join(TEMP_DIR_WAV.SUB_ONE, "sub-one-audio.wav");
+          outputPath = path.join(TEMP_DIR_WAV_CUT.SUB_ONE, "sub-one-audio.wav");
+          audioPaths.subOneAudio = { outputPath, adjustedOutputPath };
+          ensureDir(TEMP_DIR_WAV_CUT.SUB_ONE);
+          ensureDir(TEMP_DIR_WAV_ADJUSTED.SUB_ONE);
+          break;
+
+        case "subTwoStartPoint":
+          adjustedOutputPath = path.join(TEMP_DIR_WAV_ADJUSTED.SUB_TWO, "sub-two-audio.wav")
+          inputPath = path.join(TEMP_DIR_WAV.SUB_TWO, "sub-two-audio.wav");
+          outputPath = path.join(TEMP_DIR_WAV_CUT.SUB_TWO, "sub-two-audio.wav");
+          audioPaths.subTwoAudio = { outputPath, adjustedOutputPath };
+          ensureDir(TEMP_DIR_WAV_CUT.SUB_TWO);
+          ensureDir(TEMP_DIR_WAV_ADJUSTED.SUB_TWO);
+          break;
+
+        default:
+          throw new Error(`Unknown VideoLabel: ${videoLabel}`);
+      }
+
+      await cutAudio(inputPath, outputPath, startPoint);
+    }
+
+    const audioList = Object.values(audioPaths).map((paths) => paths.outputPath);
+    const adjustedStartTimes = await extractStartTime(audioList);
+    const audioLabels = Object.keys(audioPaths);
```

Comment:
민석님! 사소한 부분이지만 audioLabels, audioList 모두 audioPaths의 key, value를 받아 사용하고 있어 아래와 같이 순서 변경해주시면 가독성 부분에서 좋을 것 같다는 의견입니다 👍

```suggestion
    const audioLabels = Object.keys(audioPaths);
    const audioList = Object.values(audioPaths).map((paths) => paths.outputPath);
    const adjustedStartTimes = await extractStartTime(audioList);
```